byway/src/main/webapp/index.js

```diff
@@ -0,0 +1,16 @@
+window.onload = function(){ 
+  fetch("/login").then(response => response.json()).then((login) =>{
+  if (status == false) {
```

Comment:
What is status in this context? is this supposed to be a value from the login response?

Reply:
sorry status is meant to be .IsLoggedIn